lib/omniauth/openid_connect/version.rb

```diff
@@ -1,5 +1,5 @@
 module OmniAuth
   module OpenIDConnect
-    VERSION = "0.2.2"
+    VERSION = "0.2.4"
```

Comment:
any reason we're skipping 3?

Reply:
um no. I thought I had 3. It's a typo. thx